cpp/src/arrow/dataset/file_csv_test.cc

```diff
@@ -190,5 +208,24 @@ N/A,bar
   }
 }
 
+INSTANTIATE_TEST_SUITE_P(TestUncompressedCsv, TestCsvFileFormat,
+                         ::testing::Values(Compression::UNCOMPRESSED));
+#ifdef ARROW_WITH_BROTLI
+INSTANTIATE_TEST_SUITE_P(TestBrotliCsv, TestCsvFileFormat,
+                         ::testing::Values(Compression::BROTLI));
```

Comment:
Can you also test with bz2 (which is much more likely for compression of CSV files than Brotli)?